ports/esp32s2/boards/microdev_micro_s2/board.h

```diff
@@ -32,38 +32,38 @@
 // Enter UF2 mode if GPIO is pressed while 2nd stage bootloader indicator
 // is on e.g RGB = Purple. If it is GPIO0, user should not hold this while
 // reset since that will instead run the 1st stage ROM bootloader
-#define PIN_BUTTON_UF2        0
+#define PIN_BUTTON_UF2      0
 
 //--------------------------------------------------------------------+
 // LED
 //--------------------------------------------------------------------+
 
 // GPIO connected to Neopixel data
-#define NEOPIXEL_PIN          33
+#define NEOPIXEL_PIN        33
 
 // Brightness percentage from 1 to 255
-#define NEOPIXEL_BRIGHTNESS   0x10
+#define NEOPIXEL_BRIGHTNESS 0x10
 
 // Number of neopixels
-#define NEOPIXEL_NUMBER       1
+#define NEOPIXEL_NUMBER     1
 
 // LED for indicator and writing flash
 // If not defined neopixel will be use for flash writing instead
-#define LED_PIN               21
-#define LED_STATE_ON          1
+#define LED_PIN             21
+#define LED_STATE_ON        1
 
 //--------------------------------------------------------------------+
 // USB UF2
 //--------------------------------------------------------------------+
 
-#define USB_VID           0x239A
-#define USB_PID           0x00C5
-#define USB_MANUFACTURER  "microDev"
-#define USB_PRODUCT       "microS2"
+#define USB_VID             0x239A
+#define USB_PID             0x00C5
+#define USB_MANUFACTURER    "MicroDev"
+#define USB_PRODUCT         "microS2"
 
-#define UF2_PRODUCT_NAME  USB_MANUFACTURER " " USB_PRODUCT
-#define UF2_BOARD_ID      "microDev-microS2"
-#define UF2_VOLUME_LABEL  "MICROS2BOOT"
-#define UF2_INDEX_URL     "https://circuitpython.org/board/microdev_micro_s2"
+#define UF2_PRODUCT_NAME    USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID        USB_MANUFACTURER "-" USB_PRODUCT
```

Comment:
Board-id should follow the uf2 specs https://github.com/microsoft/uf2#files-exposed-by-bootloaders . I did miss that in the initial PR.

```
The Board-ID field is machine-readable and consists of a number of dash-separated tokens. The first token is the CPU type, second is the board type, and third is the board revision. More tokens can be also added.
```

Reply:
Now using `ESP32S2-MICROS2-V1`.